core/net/ipv6/uip-nd6.c

```diff
@@ -682,7 +684,6 @@ uip_nd6_ra_output(uip_ipaddr_t * dest)
   nd6_opt_offset = UIP_ND6_RA_LEN;
 
 
-#if !UIP_CONF_ROUTER
```

Comment:
Don't we need `#if (!UIP_CONF_ROUTER || UIP_ND6_SEND_RA)` here?
The code uses `prefix`, which is conditionally declared